test/integration/cli-domain-package-static-files-minify-file.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var expect = require('chai').expect;
+
+describe('cli : domain : package-static-files : minify-file', function(){
+	var minifyFile = require('../../src/cli/domain/package-static-files/minify-file');
+
+	describe('when minifying .js file', function(){
+
+		describe('when file contains es6', function(){
+
+			var content = 'const hi = (name) => `hello ${name}`;';
+			var minified;
+
+			before(function(){
+				minified = minifyFile('.js', content);
+			});
+
+			it('should minify it', function(){
```

Comment:
With a snapshot I would test that
```js
const hi = (name) => `hello ${name}`;
```
would always translate to
```js
var hi=function(n){return"hello "+n};
```
Which is fine in this scenario, but what if at some point I upgrade babel and this starts translating to 
```js
var hi=function(a){return"hello "+a};
```
So, my thinking was that I wanted to test something without taking the risk of coupling it too much to the snapshot. So I guess that was by design. Perhaps I'm overthinking this - as I see the benefit of seeing the simplicity of A to B transformation for a test readability point of view.